types/assert-equal-jsx/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for assert-equal-jsx 1.0
+// Project: https://github.com/thejameskyle/assert-equal-jsx
+// Definitions by: Josh Toft <https://pack.io>
```

Comment:
Would be better to use your github page (https://github.com/seryl) so we can contact you about issues/pull requests relating to this definition.